action.yml

```diff
@@ -21,11 +21,17 @@ inputs:
 runs:
   using: "composite"
   steps:
+    - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
+      with:
+        repository: nf-core/setup-nextflow
+        ref: v2.1.4
```

Comment:
I don't think that we want to do this - it means that the end user effectively can't choose which version of the action to use, right? Whatever they pick after v2.1.4 the newer version will check out the code from this older version.

In #175 I used `${{ github.action_ref }}` for this reason:

```yaml
      with:
        repository: nf-core/setup-nextflow
        ref: ${{ env.ACTION_REF }}
        path: setup-nextflow
      env:
        ACTION_REF: ${{ github.action_ref }}
```

Reply:
when bumping the version of this action, one needs to bump the ref here as well. `${{ github.action_ref }}` was empty in my test cases leading to the default branch of setup-nextflow to be checked out